docs/specification.md

```diff
@@ -681,14 +648,11 @@ Parameter (in Michelson)
 ```
 (pair %configureMinter
   (address %sender)
-  (pair
-    (address %minter)
-    (nat %tokenId)
-))
+  (address %minter)
+)
 ```
 
-- Adds `minter` to global minter list to allow him mint tokens of specific
-  token type.
+- Adds `minter` to global minter list to allow him mint tokens.
```

Comment:
@gromakovsky AFAIU adding to the global minter list does not allow him to mint tokens. For that the mint allowance has to be set to some non-zero value in the ledger as well.

Reply:
Well, below we are saying that:
> - Minter cannot mint tokens untill `set_minting_allowance`  is specified.

I think wording can be a bit improved here. Adding to the minter list is required, but not sufficient. Will improve in the next part.